ironfish-rust/src/transaction/evm.rs

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+ use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+ use std::io;
+
+use crate::errors::IronfishError;
+ 
+ #[derive(Clone)]
+ pub struct EvmDescription {
+     pub(crate) nonce: u64,
+    // TODO: gas price and limit here, or in 
+    //  pub(crate) gas_price: u64,
+    //  pub(crate) gas_limit: u64,
+     pub(crate) to: [u8; 20],
```

Comment:
nit - I think the `to` field can be empty for Ethereum transactions, like for contract deployments